android-core/src/main/java/com/mparticle/internal/ConfigManager.java

```diff
@@ -1271,6 +1271,11 @@ private void triggerMpidChangeListenerCallbacks(long mpid, long previousMpid) {
         }
     }
 
+    //keep this flag value `false` until actual implementation done
+    public boolean isFeatureFlagEnabled() {
+        return false;
+    }
+
```

Comment:
this will need to be updated right?  This is probably why the code was originally merged into the `feature-audience-api` branch initially.  so that when it was ready to go, this would be updated